test/restart-failed.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+var debug = require('debug')('strong-cluster-control:test');
+var cluster = require('cluster');
+var control = require('../');
+
+var SIZE = global.SIZE;
+
+if (cluster.isWorker) {
+  if (cluster.worker.id > SIZE)
+    process.exit(3);
+  return;
+}
+
+var tap = require('tap');
+
+tap.test('good workers are not killed', function(t) {
+  var ended;
+
+  control.start({size: SIZE});
+
+  function size() {
+    return Object.keys(cluster.workers).length;
+  }
+
+  control.once('resize', function() {
+    if (ended) return;
```

Comment:
Restart, but only once, and only if the test hasn't already ended by then? This handler has no assertions in it, is it for working around some sort of edge case?